src/FrequencyMapAgent.cpp

```diff
@@ -320,9 +362,10 @@ namespace geopm
     std::vector<std::string> FrequencyMapAgent::policy_names(void)
     {
 
-        std::vector<std::string> names{"FREQ_DEFAULT", "FREQ_UNCORE"};
+        std::vector<std::string> names{"FREQ_DEFAULT", "FREQ_UNCORE", "FREQ_GPU_DEFAULT"};
```

Comment:
A user might assume "FREQ_DEFAULT" defines a default frequency for any frequency controls covered by this agent.

Maybe `"FREQ_CPU_CORE_DEFAULT", "FREQ_CPU_UNCORE_DEFAULT", "FREQ_GPU_CORE_DEFAULT"`?